internal/tools/pipeline/providers/cms/pipeline_cms.go

```diff
@@ -352,7 +352,18 @@ func (c *pipelineCm) BatchGetConfigs(ctx context.Context, req *pb.CmsNsConfigsBa
 		}(config)
 	}
 	wait.Wait()
-	return newConfigs, nil
+	var sortedConfigs []*pb.PipelineCmsConfig
```

Comment:
Abstract a method.

And how about duplicated items among namespaces?

Reply:
duplicated items are sorted by namespaces